lib/member.rb

```diff
@@ -0,0 +1,11 @@
+require 'scraped'
+
+class Member < Scraped::HTML
+  field :section do
+    @section ||= MemberSection.new(response: response, noko: noko)
```

Comment:
I think the `@section ||=` bit here (and the `@page` version below) are probably unnecessary optimizations at this stage. Probably best to drop them for now until they prove they are needed.